src/preact/base-element.js

```diff
@@ -192,7 +306,7 @@ export class PreactBaseElement extends BaseElement {
     /** @private {?Array} */
     this.contextValues_ = null;
 
-    /** @private {?Node} */
+    /** @type {Element | null} */
```

Comment:
jw, do you prefer `|null` over `?Element`?

Reply:
Not really, I just saw that `TS` doesn't have a shorthand..and `?` in TS implies `|undefined`. This leads me to prefer explicit `|null` to save the confusion while we switch